Update link to speedtestservers

LEJOUI opened the pull request:
The link to the speedtestservers seems to be broken. The new link does work. Also the server_id seems to be mandatory at this point. So this sets the server_id to be required and in the example configuration.yaml inserts an example server_id.

**Description:**


**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

LEJOUI committed:
Update link to speedtestservers and server_id 

The link to the speedtestservers seems to be broken. The new link does work. Also the server_id seems to be mandatory at this point. So this sets the server_id to be required and in the example configuration.yaml inserts an example server_id.

frenck commented on source/_integrations/speedtestdotnet.markdown:
```diff
@@ -43,7 +43,7 @@ monitored_conditions:
 server_id:
   description: Specify the speed test server to perform the test against.
-  required: false
+  required: true
```
This option is not required.

Code reference:
<https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/speedtestdotnet/__init__.py#L26>

LEJOUI replied:
The code indeed states it's not required. I'll create an issue about this instead. Speedtest.net is somehow not working over here without server_id

frenck replied:
That is fine, but it still means this change has to be reverted right now.

frenck reviewed (changes requested):
Hi there @LEJOUI, thanks for opening a PR!

Welcome to the Home Assistant Community 🎉 

I've reviewed it and left a comment. Could you please take a look? Awesome! 👍

LEJOUI committed:
Removes sever_id from configuration.yaml

Removes sever_id from configuration.yaml. It's not necessary if server_id is optional.

LEJOUI committed:
Changes server_id to be optional again

frenck commented:
Thanks, @LEJOUI! 👍 

Happy Hacktoberfest 🎉
